api.swagger.yaml

```diff
@@ -14511,6 +15150,124 @@ components:
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+                name:
+                  description: 'The name of the query category.'
+                  type: string
+                  example: 'Important'
+            author:
+              description: 'The author of this query.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/user-trimmed-schema'
+            timestamp_created:
+              description: 'This is the timestamp (in seconds) of when the query was created.'
+              type: integer
+              example: 1510941596
+            timestamp_updated:
+              description: 'This is the timestamp (in seconds) of when the query was last updated.'
+              type: integer
+              example: 1510941696
+            comment_count:
+              description: 'Count of comments on the query.'
+              type: integer
+              example: 5
+            likes_count:
+              description: 'Count of likes on the query.'
+              type: integer
+              example: 5
+    query-schema-common:
+      type: object
+      properties:
+        title:
+          description: 'Title of the query.'
+          type: string
+          example: 'Company talent competition'
+        body:
+          description: 'The body of the query.'
+          type: string
+          example: 'Demonstrate your talent at the company talent competition.'
+        query_reference:
+          description: 'The query reference.'
+          type: string
+          example: 'Q123'
+        review_date_timestamp:
+          description: 'The review date timestamp for the query (in seconds).'
+          type: integer
+          example: 1510941596
+    support-ticket-schema:
+      allOf:
+        - $ref: '#/components/schemas/support-ticket-schema-common'
+        - type: object
+          properties:
+            uuid:
+              description: 'Invotra UUID of the support ticket.'
+              type: string
+              format: uuid
+              example: '01234567-89ab-cdef-1234-56789abcdef0'
+            author:
+              description: 'The author of this support ticket.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/user-trimmed-schema'
+            timestamp_created:
+              description: 'This is the timestamp (in seconds) of when the support ticket was created.'
+              type: integer
+              example: 1510941596
+            timestamp_updated:
+              description: 'This is the timestamp (in seconds) of when the support ticket was last updated.'
+              type: integer
+              example: 1510941696
+            comment_count:
+              description: 'Count of comments on the support ticket.'
+              type: integer
+              example: 5
+            likes_count:
```

Comment:
Support tickets don't have likes, can you see this available in solr? I was checking in the drupal UI

Reply:
done